pinot-test-container/src/main/java/org/apache/pinot/testcontainer/PinotContainer.java

```diff
@@ -139,7 +172,10 @@ public void addTables() throws IOException, InterruptedException {
 
   private long getTableSize(final String tableName) throws IOException, InterruptedException {
     final ExecResult checkIngestionIsFinished = this.execInContainer(
-        "curl", "-X", "GET", String.format("http://localhost:9000/tables/%s_OFFLINE/size?detailed=false", tableName)
+        "curl",
```

Comment:
it seems we don't have the same code style configuration.
let's loop with Suvodeep and Shounak to ensure everybody has the same.

Reply:
not a blocker of course

Reply:
🤷 I imported the styles from https://github.com/startreedata/thirdeye/blob/master/thirdeye-checkstyle.xml, but ya Intellij still didn't like this line.